mne/channels/channels.py

```diff
@@ -440,18 +440,36 @@ def inst_has(attr):
 class InterpolationMixin(object):
     """Mixin class for Raw, Evoked, Epochs
     """
-    def interpolate_bads_eeg(self):
-        """Interpolate bad channels
 
-        Operates in place.
+    def interpolate_bads(self, reset_bads=True):
```

Comment:
On Tue, Apr 28, 2015 at 4:20 PM, Alexandre Gramfort <
notifications@github.com> wrote:

> In mne/channels/channels.py
> <https://github.com/mne-tools/mne-python/pull/2011#discussion_r29246868>:
>
> >
> > -        Operates in place.
> > +    def interpolate_bads(self, reset_bads=True):
>
>  but isn't this what is done in most epoch methods?
> if you think that you might want to chain methods return self. but don't
> write: evoked_interp = evoked.interpolate_bads() if interpolate_bads has no
> copy parameter and operates in place.
>

ok got it :)


> Sorry I do what I can to review from my phone...
>

no problem :) I'll bug you till I get it ;)

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2011/files#r29246868>.
>